doc/sphinx/proof-engine/tactics.rst

```diff
@@ -3153,14 +3153,15 @@ the conversion in hypotheses :n:`{+ @ident}`.
      use the name of the constant the (co)fixpoint comes from instead of
      the (co)fixpoint definition in recursive calls.
 
-   The ``cbn`` tactic is claimed to be a more principled, faster and more
-   predictable replacement for ``simpl``.
+   The :tacn:`cbn` tactic is claimed to be a more principled, faster and more
+   predictable replacement for :tacn:`simpl`.
 
-   The ``cbn`` tactic accepts the same flags as ``cbv`` and ``lazy``. The
-   behavior of both ``simpl`` and ``cbn`` can be tuned using the
-   Arguments vernacular command as follows:
+   The :tacn:`cbn` tactic accepts the same flags as :tacn:`cbv` and
+   :tacn:`lazy`. The behavior of both :tacn:`simpl` and :tacn:`cbn`
+   can be tuned using the Arguments vernacular command as follows:
```

Comment:
```suggestion
   can be tuned using the :cmd:`Arguments <Arguments (implicits)>` vernacular command as follows:
```

Reply:
I don't commit this suggestion because `Arguments` is documented in many places but this particular use case is not documented elsewhere. Future work: rework this part so that it is also referenced in the command index.